test/stdlib/Reflection_objc.swift

```diff
@@ -12,6 +12,7 @@
 //
 
 // XFAIL: linux
+// UNSUPPORTED: OS=windows-msvc
```

Comment:
Is this UNSUPPORTED still supposed to be here?

Reply:
I am torn.  Is it UNSUPPORTED or XFAIL?  The problem is that this is an executable test, and does need an ObjC runtime ...

Reply:
That sounds like `REQUIRES: objc_interop`, then.

Reply:
I don't see much that isn't ObjC interop testing?  Mind having a QuickLook (ha!)?

Reply:
Arguably a bunch of that should work with corelibs-foundation too, but yeah, I think changing to `REQUIRES: objc_interop` would be fine here, since we have the regular Reflection.swift now.

Reply:
Yeah, I want to eventually enable a bunch fo the tests that we have that use Foundation to use s-c-f.